gui/mozregui/report.py

```diff
@@ -60,11 +61,33 @@ def status_text(self):
         return msg % self.state_text
 
 
+def bulk_action_slots(slots, signal_object, slot_object, action):
+    if signal_object and slot_object:
+        for name in slots:
+            signal = getattr(signal_object, name)
+            slot = getattr(slot_object, name)
+            if action == "connect":
```

Comment:
you can also use a getattr here:
```
getattr(signal, action)(slot)
```
would do the job in one line. Well maybe a bit less readable.